MAUI/Kanban-Board/Cards.md

```diff
@@ -42,83 +42,83 @@ You can replace the entire card template with your own design using [`CardTempla
 
 {% highlight xaml %}
 
-    <kanban:SfKanban.CardTemplate >
+<kanban:SfKanban.CardTemplate >
 
-    <DataTemplate>
-        <StackLayout WidthRequest="250" Orientation="Vertical" BackgroundColor="Gray" Padding="10,10,10,10"> 
-            
-            <StackLayout  Orientation="Horizontal"> 
+<DataTemplate>
+    <StackLayout WidthRequest="250" Orientation="Vertical" BackgroundColor="Gray" Padding="10,10,10,10"> 
+        
+        <StackLayout  Orientation="Horizontal"> 
 
-                <Label Text="{Binding Path=Title}" TextColor="Silver" HorizontalOptions="StartAndExpand" >
-                </Label>
+            <Label Text="{Binding Path=Title}" TextColor="Silver" HorizontalOptions="StartAndExpand" >
```

Comment:
@EzhilarasanElangovan31 we can set Title directly. Is any need to mention as Path=Title?

Reply:
Updated